tests/ALCProxy.Tests/ALCProxy.Tests.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <!-- GenerateProgramFile>false</GenerateProgramFile -->
+    <IsPackable>false</IsPackable>
+
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="2.0.30" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="xunit" Version="2.4.0" />
+    <PackageReference Include="xunit.performance.api" Version="1.0.0-beta-build0020" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0">
```

Comment:
Generally, we import common.prop/dependencies.props and use the version specified there: https://github.com/dotnet/corefxlab/blob/58b32dabce427f7ef37a7ff9bdbb3c64321229d5/tools/dependencies.props#L2-L19